buildchain/buildchain/versions.py

```diff
@@ -364,200 +425,65 @@ def _version_prefix(version: str, prefix: str = 'v') -> str:
 
 
 DEB_PACKAGES = (
-    Package(
+    PackageVersion(
```

Comment:
Save some screen real estate and squash into a single line: `PackageVersion(name='containerd'),` (ditto for the others)

Reply:
Done in a following commit